.github/workflows/master.yml

```diff
@@ -45,15 +45,21 @@ jobs:
         writeLines(sprintf("R-%i.%i", getRversion()$major, getRversion()$minor), ".github/R-version")
       shell: Rscript {0}
 
+    - name: Get OS name
+      id: os-name
+      run: |
+        # `os_name` looks like "Ubuntu-20.04.1-LTS"
+        os_name=$(lsb_release -ds | sed 's/\s/-/g')
+        echo "::set-output name=os-name::$os_name"
     - name: Cache R packages
       if: runner.os != 'Windows'
       uses: actions/cache@v2
       with:
         path: ${{ env.R_LIBS_USER }}
         # We cache R dependencies based on a tuple of the current OS, the R version, and the list of
         # R dependencies
-        key: ${{ runner.os }}-${{ hashFiles('.github/R-version') }}-1-${{ hashFiles('.github/depends.Rds') }}
-        restore-keys: ${{ runner.os }}-${{ hashFiles('.github/R-version') }}-1-
+        key: ${{ steps.os-name.outputs.os-name }}-${{ hashFiles('.github/R-version') }}-1-${{ hashFiles('.github/depends.Rds') }}
+        restore-keys: ${{ steps.os-name.outputs.os-name }}-${{ hashFiles('.github/R-version') }}-1-
```

Comment:
Update the cache key to refresh the cache when Ubuntu version is updated.

Reply:
The R check was failing with:

https://github.com/mlflow/mlflow/runs/1819742999#step:10:496

```
── 1. Error: (unknown) (@test-model-xgboost.R#14)  ─────────────────────────────
unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
```